_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -88,8 +88,107 @@ Option | Required | Type | Description
+* `opensearch_action` - A default value for mapping DynamoDB event actions to OpenSearch actions. This action will be `index` for export items, and `INSERT` or `MODIFY` for stream events, and `REMOVE` stream events when the OpenSearch action is `delete`.
+* `dynamodb_event_name` - The exact event type for the item. Will be `null` for export items, and either `INSERT`, `MODIFY`, or `REMOVE` for stream events.
+* `table_name` - The DynamoDB table name that an event came from.
+
+
+## Permissions
+
+The following shows the minimum required permissions for running DynamoDB as a source:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "allowDescribeTable",
+        "Effect": "Allow",
+        "Action": [
+          "dynamodb:DescribeTable"
+        ],
+        "Resource": [
+          "arn:aws:dynamodb:us-east-1:{account-id}:table/my-table"
+        ]
+      },
+       {
+            "Sid": "allowRunExportJob",
+            "Effect": "Allow",
+            "Action": [
+                "dynamodb:DescribeContinuousBackups",
+                "dynamodb:ExportTableToPointInTime"
+            ],
+            "Resource": [
+                "arn:aws:dynamodb:us-east-1:{account-id}:table/my-table"
+            ]
+        },
+        {
+            "Sid": "allowCheckExportjob",
+            "Effect": "Allow",
+            "Action": [
+                "dynamodb:DescribeExport"
+            ],
+            "Resource": [
+                "arn:aws:dynamodb:us-east-1:{account-id}:table/my-table/export/*"
+            ]
+        },
+        {
+            "Sid": "allowReadFromStream",
+            "Effect": "Allow",
+            "Action": [
+                "dynamodb:DescribeStream",
+                "dynamodb:GetRecords",
+                "dynamodb:GetShardIterator"
+            ],
+            "Resource": [
+                "arn:aws:dynamodb:us-east-1:{account-id}:table/my-table/stream/*"
+            ]
+        },
+        {
+            "Sid": "allowReadAndWriteToS3ForExport",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:AbortMultipartUpload",
+                "s3:PutObject",
+                "s3:PutObjectAcl"
+            ],
+            "Resource": [
+                "arn:aws:s3:::my-bucket/*"
+            ]
+        }
+    ]
+}
+```
+
+When performing an export, `"Sid": "allowReadFromStream"` section is not required. If only reading from DynamoDB streams, the 
+`"Sid": "allowReadAndWriteToS3ForExport"`, `"Sid": "allowCheckExportjob"`, ` "Sid": "allowRunExportJob"` sections are not required.
+
+## Metrics
+
+The `dynamodb` source includes the following metrics:
 
+### Counters
 
+* `exportJobSuccess` - The number of export jobs that have been submitted successfully.
```

Comment:
The list items should all be separated from their descriptions by either a colon (`partition_key`: The) or an en dash (`partition_key` -- The).